.github/workflows/spark_master_test.yaml

```diff
@@ -9,11 +9,11 @@ jobs:
         # These Scala versions must match those in the build.sbt
         scala: [2.13.13]
         # Important: This list of shards must be [0..NUM_SHARDS - 1]
-        shard: [0, 1, 2]
+        shard: [0, 1, 2, 3]
```

Comment:
just curious: why this change? e.g. have we validated that increasing the shards from 3 to 4 improves things? e.g. if so, I'm curious why it was set to 3 instead of 4 in the first place?

Reply:
Not sure why it was set to 3. I thought they were kind of slow so updated them to match master.